### PR TITLE
Header enhancements

### PR DESCRIPTION
**Pull request by TiffanyNg-okta**
## Description:
- update pink color for active state text in header
- hide search placeholder when its full text doesn't fit within a smaller search box
- unhide and style clear button in search input
- add expand/collapse for "Community" in mobile header nav

### Resolves:

* [OKTA-355565](https://oktainc.atlassian.net/browse/OKTA-355565)

### CODE REVIEW

**Comment by TiffanyNg-okta** [comment]
@shawyu-okta @francoislehoux-okta I'm having git access issues so it seems I can't add reviewers or tag devdocs team, I'll try again later but can you start reviewing when you have some time? Thanks!

**Comment by shawyu-okta** [comment]
@TiffanyNg-okta can you please make a Netlify deploy with the feature flag on and post the link here?

**Review by susanharper-okta — APPROVED** [review]
👍

**Comment by TiffanyNg-okta** [comment]
https://5ff7ce5cde870717d6d53736--reverent-murdock-829d24.netlify.app/pricing/

File: packages/@okta/vuepress-theme-prose/assets/css/components/_searchBar.scss [hunk 1]
```diff
@@ -66,6 +76,12 @@ $redesign_placeholder_color: #616161;
             color: $redesign_placeholder_color;
           }
 
+          @media (max-width: 1080px) {
+            &::placeholder {
+              color: transparent;
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-theme-prose/assets/css/components/_searchBar.scss:81]: Is there any way we can combine this with the next rule applying to the same pseudo-element? They seem overlapping.

File: packages/@okta/vuepress-theme-prose/index.js [hunk 2]
```diff
@@ -14,9 +14,9 @@ module.exports = (options, ctx) => {
       $page.fullHeaders = resolveHeaders($page);
 
       // Set to true to enable the redesign in development
-      const redesign = process.env.REDESIGN || false;
+      //const redesign = process.env.REDESIGN || false;
       // Respect the redesign flag unless this is a production build
-      $page.redesign = !ctx.isProd && redesign;
+      $page.redesign = true;
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-theme-prose/index.js:19]: Just a reminder note to revert before merging

File: packages/@okta/vuepress-theme-prose/assets/css/components/_header.scss [hunk 3]
```diff
@@ -249,6 +253,21 @@ header.redesign {
...
+              transition: transform 300ms ease-in-out;
+            }
+
+            &.open {
+              > ul.submenu--items {
+                max-height: 100%;
+              }
+              
+              > span.link:after {
+              transform: rotate(0deg);
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-theme-prose/assets/css/components/_header.scss:268]: nit:
```suggestion
                transform: rotate(0deg);
```

File: packages/@okta/vuepress-theme-prose/components/MenuItem.vue [hunk 4]
```diff
@@ -56,8 +58,13 @@ export default {
...
     }
   },
+  data() {
+    return {
+      isOpen: false,
+    }
+  },
   computed: {
-    expanded() {
+    isExpandable() {
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-theme-prose/components/MenuItem.vue:67]: 👍 on the rename

Comment (TiffanyNg-okta) [reply packages/@okta/vuepress-theme-prose/components/MenuItem.vue:67]: thanks!

**Review by shawyu-okta — APPROVED** [review]
lgtm

🐻

File: packages/@okta/vuepress-theme-prose/assets/css/components/_searchBar.scss (hunk 1 above)
Comment (TiffanyNg-okta) [reply packages/@okta/vuepress-theme-prose/assets/css/components/_searchBar.scss:81]: Actually the 1080px would cover the <desktop (1024px) so I could just replace it.

**Pull request merged**